README.md

```diff
@@ -63,10 +63,43 @@ If this does not work, see the instructions for running Cadence Server at https:
 
 ## Register the Domain
 
-To register the domain, run the follownig command once before running any samples:
+To register the *sample* domain, run the following command once before running any samples:
 
     ./gradlew -q execute -PmainClass=com.uber.cadence.samples.common.RegisterDomain
 
+## See Cadence UI
+
+The Cadence Server running in a docker container includes web UI.
+
+Connect to [http://localhost:8088](http://localhost:8088)
+
+Enter the *sample* domain. You'll see "No Results" page. After running any sample change the filter in the
```

Comment:
You'll see "No Results"... -> You'll see a "No Results"...
Please add a comma after "any sample"